File: benchmarks/node/strings.js
const parts = [];
for (let i = 0; i < 200000; i++) parts.push("word" + (i % 10) + " ");
const text = parts.join("").trim();
const words = text.split(" ");
const joined = words.join("-");
console.log(text.length, words.length, joined.length);

File: benchmarks/rust/strings.rs
fn main() {
    let mut text = String::new();
    for i in 0..200_000 {
        text.push_str("word");
        text.push_str(&(i % 10).to_string());
        text.push(' ');
    }
    let trimmed = text.trim().to_string();
    let words: Vec<&str> = trimmed.split(' ').collect();
    let joined = words.join("-");
    println!("{} {} {}", trimmed.len(), words.len(), joined.len());
}
